packages/config/config/devices/0x015f/mh-s220.json

```
@@ -0,0 +1,177 @@
+{
+	"manufacturer": "McoHome Technology Co., Ltd.",
+	"manufacturerId": "0x015f",
+	"label": "MH-S220",
+	"description": "Micro Switch",
+	"devices": [
+		{
+			"productType": "0x2201",
+			"productId": "0x1252"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Switch 1",
+			"maxNodes": 5
+		},
+		"2": {
+			"label": "Switch 2",
+			"maxNodes": 5
+		},
+		"3": {
+			"label": "Lifeline",
+			"maxNodes": 1,
+			"isLifeline": true
+		}
+	},
+	"paramInformation": [
+		{
+			"#": "1",
+			"$import": "~/templates/master_template.json#state_after_power_failure_off_prev"
+		},
+		{
+			"#": "2",
+			"label": "External Switch Type",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Momentary Buttons",
+					"value": 0
+				},
+				{
+					"label": "2-State Switches",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "3",
+			"label": "2-State Switches",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Toggle Switch without specified on/off position",
+					"value": 0
+				},
+				{
+					"label": "Toggle Switch with specified on/off position",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "4",
+			"label": "Switch 1 - Operation Report to Association Group 1",
```

```suggestion
			"label": "Switch 1: Reports to Association Group 1",
```
for consistency with other files please use `:` instead of `-`.
Same for the other parameters